openedx/core/djangoapps/user_authn/views/login.py

```diff
@@ -211,10 +213,16 @@ def _authenticate_first_party(request, unauthenticated_user):
     username = unauthenticated_user.username if unauthenticated_user else ""
 
     try:
-        return authenticate(
-            username=username,
-            password=request.POST['password'],
-            request=request)
+        if PASSWORD_UNICODE_NORMALIZE.is_enabled():
+            return authenticate(
+                username=username,
+                password=normalize_password(request.POST['password']),
+                request=request)
+        else:
+            return authenticate(
+                username=username,
+                password=request.POST['password'],
+                request=request)
```

Comment:
Optional nit: I might rewrite this to keep the logic written once, and just change the data you care about:
```
        password = request.POST['password']
        if PASSWORD_UNICODE_NORMALIZE.is_enabled():
            password = normalized()
        return authenticate(
                username=username,
                password=password,
                request=request)
```